package.json

```diff
@@ -36,7 +39,6 @@
     "ts-jest": "^24.0.2",
     "ts-loader": "^6.2.0",
     "tslib": "^1.4.0",
-    "tslint": "^5.20.0",
```

Comment:
The `npm run lint` script still tries to invoke `tslint`

Reply:
ah!